src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    protected String description;
+    protected boolean isCompleted;
+
+    public Task(String description) {
+        this.description = description;
+        this.isCompleted = false;
+    }
+
+    public String getStatusIcon() {
+        return (isCompleted ? "✔" : "X"); // mark done task with X
```

Comment:
may be bit confusing because all the boxes will be filled, making it hard for users to see exactly which task is completed or not at a glance